.github/workflows/codeql-workflow.yml

```diff
@@ -0,0 +1,42 @@
+name: "CodeQL"
+
+on:
+  workflow_dispatch:
```

Comment:
Not sure there is a need for `workflow_dispatch`, apart from testing purposes. Do you want to keep it?

Reply:
Yep thanks, remove in last version